lib/hmmlearn/_hmmc.pyx

```diff
@@ -23,6 +23,8 @@ cdef inline dtype_t _max(dtype_t[:] X) nogil:
 
 cdef inline dtype_t _logsumexp(dtype_t[:] X) nogil:
     cdef dtype_t X_max = _max(X)
+    cdef ssize_t i
```

Comment:
unneeded?

Reply:
I added it for consistency. In all other functions the iteration variables are declared as well.

Reply:
Oh, I see.  Can you make that a separate commit to keep things self-contained(ish)?  I can do it directly, too.